### PR TITLE
chore: add nodeSourcemap config prop to README

### PR DESCRIPTION
**Pull request by eduardoboucas**
**- Summary**

The `nodeSourcemap` configuration property was missing from the READMe. This PR fixes that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,13 +105,23 @@ The following properties are accepted:
 
 - `nodeBundler`
 
-  - _Type_: `string`
-  - _Default value_: `zisi`
+  - _Type_: `boolean`
+  - _Default value_: `false`
```
Comment (erezrokah) [inline README.md:109]:
```suggestion
  - _Type_: `string`
  - _Default value_: `zisi`
```

File: README.md [hunk 2]
```diff
@@ -105,13 +105,23 @@ The following properties are accepted:
...
 
   The bundler to use when processing JavaScript functions. Possible values: `zisi`, `esbuild`, `esbuild_zisi`.
 
   When the value is `esbuild_zisi`, `esbuild` will be used with a fallback to `zisi` in case of an error.
 
+- `nodeSourcemap`
+
+  - _Type_: `string`
+  - _Default value_: `zisi`
```
Comment (erezrokah) [inline README.md:118]:
```suggestion
  - _Type_: `boolean `
  - _Default value_: `false`
```

File: README.md [hunk 3]
```diff
@@ -105,13 +105,23 @@ The following properties are accepted:
...
+
+  Whether to include a [sourcemap file](https://developer.mozilla.org/en-US/docs/Tools/Debugger/How_to/Use_a_source_map)
+  in the generated archive.
+
+  Available only when `nodeBundler` is seet to `esbuild` or `esbuild_zisi`.
```
Comment (erezrokah) [inline README.md:123]:
```suggestion
  Available only when `nodeBundler` is set to `esbuild` or `esbuild_zisi`.
```

**Review by erezrokah — CHANGES_REQUESTED** [review]
I think some changes were shifted to the wrong location.

Also not sure about the other changes

**Comment by eduardoboucas** [comment]
> I think some changes were shifted to the wrong location.
> 
> Also not sure about the other changes

Sheesh! I made a proper mess out of this PR. No idea what happened here.

I think I fixed everything in https://github.com/netlify/zip-it-and-ship-it/pull/592/commits/2ecebe22d8e1fb6e9fd8e096619dc6971e132c40.

As for the other changes, it's just changing where the lines break. Prettier did that automatically. No idea if anything changed in the line length config? 🤔

**Review by erezrokah — APPROVED** [review]

**Comment by erezrokah** [comment]
> As for the other changes, it's just changing where the lines break. Prettier did that automatically. No idea if anything changed in the line length config? 🤔

Not sure, maybe we never ran it on this file?

**Pull request merged**